test/mocks/environment.js

```diff
@@ -81,10 +81,14 @@ export const HelixMockContentSources = {
  * @param {string} location The location to mock
  */
 export function mockLocation(document, location) {
-  const input = document.createElement('input');
-  input.id = 'sidekick_test_location';
-  input.value = location;
-  document.body.appendChild(input);
+  let input = document.getElementById('sidekick_test_location');
+  if (!input) {
+    input = document.createElement('input');
+    input.setAttribute('type', 'hidden');
```

Comment:
Any specific reason you want to hide it?

Reply:
Doesn't need to be hidden, I just gave it a type, because for some reason it was making trouble when it didn't have a type.